addons/account/models/chart_template.py

```diff
@@ -1002,3 +1016,26 @@ def _load_translations(self, langs=None, companies=None):
                                     xml_id = f"account.{company.id}_{_xml_id}"
                                     translation_importer.model_translations[mname][fname][xml_id][lang] = value
         translation_importer.save(overwrite=False)
+
+    def _load_account_name_code_translations(self, accounts):
+        """Loads translations for the accounts passed as argument from the code translations of the 'account' module.
+
+        :param accounts: the accounts to load the translations for
+        :type accounts: Model<account.account>
+        """
+
+        langs = [code for code, _name in self.env['res.lang'].get_installed()]
+        translations = {
+            lang_code: code_translations.get_python_translations('account', lang_code)
+            for lang_code in langs
+        }
+
+        for account in accounts:
+            name_translations = {
+                lang_code: translations[lang_code][account.name]
+                for lang_code in langs
+                # There are no translations for language 'en_US' (since it is the original language and not a translation)
+                if lang_code != 'en_US' and account.name in translations[lang_code]
+            }
+            if name_translations:
+                account.update_field_translations('name', name_translations)
```

Comment:
With my other suggestions, you can just feed the data from `get_python_translations` into the `TranslationImporter` of `_load_translations` and do that in batch